site/app/templates/admin/LateDays.twig

```diff
@@ -81,10 +81,26 @@
     {% endif %}
 </div>
 <script>
+    const students_and_late_days = JSON.parse(`{
+    {% for user in users %}
+        "{{ user.getId() }}":"{{ user.getAllowedLateDays() }}",
+    {% endfor %}
+    "":""
```

Comment:
Is `"":""` used for anything specific? If there are no users in the latedays granted list 
JSON.parse("{}") is  a valid default

Reply:
`JSON.parse()` doesn't like dangling commas for some reason.  Since the previous statement adds a comma after each value, I added a placeholder at the end which does not have the trailing comma.